api/models/build.js

```diff
@@ -80,25 +80,6 @@ function completeJob(err) {
     .then(() => build));
 }
 
-function toJSON() {
-  const object = Object.assign({}, this.get({
-    plain: true,
-  }));
-
-  object.createdAt = object.createdAt.toISOString();
-  object.updatedAt = object.updatedAt.toISOString();
-  if (object.completedAt) {
-    object.completedAt = object.completedAt.toISOString();
-  }
-  Object.keys(object).forEach((key) => {
-    if (object[key] === null) {
-      delete object[key];
-    }
-  });
-  delete object.token;
```

Comment:
Looks like we may still need to find a way to make sure that `toJSON` calls don't include the token.

Reply:
Ah, I see that you moved this.